.github/workflows/hadolint.yml

```diff
@@ -0,0 +1,35 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
+
+name: "Docker"
+
+on:
+    pull_request:
+        paths:
+        - "**/Dockerfile"
+        - ".hadolint.yaml"
+    push:
+        branches:
+        - "main"
+        paths:
+        - "**/Dockerfile"
+        - ".hadolint.yaml"
+
+permissions:
+    contents: "read"
+
+concurrency:
+    group: "${{ github.workflow }}-${{ github.ref }}"
+    cancel-in-progress: true
+
+jobs:
+    hadolint:
```

Comment:
Should we add this to `style.yml`?

Reply:
I name actions based on what it cares about, not base on what it does.
E.g. "style" is a job in "back-end".
https://github.com/szepeviktor/byte-level-care/tree/master/.github/workflows

Reply:
If you organize workflows based on the activity you will end up with 5 totally different targets in "style.yml".

Reply:
First let's see its output!

Reply:
I approved the run.

Reply:
Here is the output: https://github.com/szepeviktor/cachewerk_relay/actions/runs/5732711289/job/15536150993

- What rules (e.g `DL3040`) to you want to be corrected?
- What rules do you want to be ignored?

Reply:
Here is the output sorted and grouped by rule code
https://gist.github.com/szepeviktor/16c82e95c3b08906b297a2262728a2bc

Reply:
That's a lot of issues and they are pretty strict.

Reply:
That's the point of my work: prevent **future** problems.